source/jormungandr/jormungandr/interfaces/v1/Calendars.py

```diff
@@ -119,8 +74,9 @@ def __init__(self):
                                 help='The datetime considered as "now". Used for debug, default is '
                                      'the moment of the request. It will mainly change the output '
                                      'of the disruptions.')
-        self.collection = 'calendars'
-        self.collections = calendars
+        # TODO : [NAVP-805] Collection(s) : useful or useless ?
+        # self.collection = 'calendars'
+        # self.collections = api.CalendarsSerializer
```

Comment:
If it's commented out, I would think this is useless, and I would rather remove that code.
If it's useful, that will come back to us anyway ;)